specification/support/resource-manager/Microsoft.Support/preview/2019-05-01-preview/support.json

```diff
@@ -0,0 +1,1487 @@
+                "technicalTicketDetails": {
+                    "$ref": "#/definitions/TechnicalTicketDetails",
+                    "description": "Additional ticket details associated with a technical support ticket request. "
+                },
+                "quotaTicketDetails": {
+                    "$ref": "#/definitions/QuotaTicketDetails",
+                    "description": "Additional ticket details associated with a quota support ticket request. "
+                }
+            }
+        },
+        "ContactProfile": {
+            "description": "Contact information associated with support ticket.",
+            "required": [
+                "firstName",
+                "lastName",
+                "preferredContactMethod",
+                "primaryEmailAddress",
+                "preferredTimeZone",
+                "country",
+                "preferredSupportLanguage"
+            ],
+            "type": "object",
+            "properties": {
+                "firstName": {
+                    "description": "First name.",
+                    "type": "string"
+                },
+                "lastName": {
+                    "description": "Last name.",
+                    "type": "string"
+                },
+                "preferredContactMethod": {
+                    "description": "Preferred contact method.",
+                    "enum": [
+                        "email",
+                        "phone"
+                    ],
+                    "type": "string",
+                    "x-ms-enum": {
+                        "name": "PreferredContactMethod",
+                        "modelAsString": true
+                    }
+                },
+                "primaryEmailAddress": {
+                    "description": "Primary email address.",
+                    "type": "string"
+                },
+                "additionalEmailAddresses": {
+                    "description": "Additional email addresses.",
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "phoneNumber": {
+                    "description": "Phone number. This is required if preferred contact method is phone.",
+                    "type": "string"
+                },
+                "preferredTimeZone": {
+                    "description": "Time zone of the user. This is the System.TimeZoneInfo.Id value",
+                    "type": "string"
+                },
+                "country": {
+                    "description": "Country of the user. This is the ISO Alpha-3 code",
+                    "type": "string"
+                },
+                "preferredSupportLanguage": {
+                    "description": "Language of the user. This is the standard country-language code.",
+                    "type": "string"
+                }
+            }
+        },
+        "UpdateContactProfile": {
```

Comment:
Are these exposed in an existing api? Have they been cleared by privacy/GDPR?

Reply:
@KrisBash - These are already exposed in existing API. Attaching screenshot.

![image](https://user-images.githubusercontent.com/12116126/70364962-d3973900-1843-11ea-873a-1d81c05e1792.png)